libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -769,6 +769,36 @@ singleton AP_Mount method get_angle_target boolean uint8_t'skip_check float'Null
 singleton AP_Mount method get_location_target boolean uint8_t'skip_check Location'Null
 singleton AP_Mount method set_attitude_euler void uint8_t'skip_check float'skip_check float'skip_check float'skip_check
 
+userdata mavlink_camera_information_t depends AP_CAMERA_ENABLED
+userdata mavlink_camera_information_t field time_boot_ms uint32_t'skip_check read write
+userdata mavlink_camera_information_t field firmware_version uint32_t'skip_check read write
+userdata mavlink_camera_information_t field focal_length float'skip_check read write
+userdata mavlink_camera_information_t field sensor_size_h float'skip_check read write
+userdata mavlink_camera_information_t field sensor_size_v float'skip_check read write
+userdata mavlink_camera_information_t field flags uint32_t'skip_check read write
+userdata mavlink_camera_information_t field resolution_h uint16_t'skip_check read write
+userdata mavlink_camera_information_t field resolution_v uint16_t'skip_check read write
+userdata mavlink_camera_information_t field cam_definition_version uint16_t'skip_check read write
+userdata mavlink_camera_information_t field vendor_name'array 32 uint8_t'skip_check read write
+userdata mavlink_camera_information_t field model_name'array 32 uint8_t'skip_check read write
+userdata mavlink_camera_information_t field lens_id uint8_t'skip_check read write
+userdata mavlink_camera_information_t field cam_definition_uri'array 140 uint8_t'skip_check read write
+userdata mavlink_camera_information_t field gimbal_device_id uint8_t'skip_check read write
```

Comment:
we should bring in the XML for camera_id

Reply:
@tridge: Is this PR blocked on getting the MAVLink XML changes for `camera_id` brought in?